irods_specific_implementation/irods_rest_logical_path_delete_api_implementation.h

```diff
@@ -0,0 +1,88 @@
+#include "irods_rest_api_base.h"
+
+#include "constants.hpp"
```

Comment:
Add an empty line after this header.

This keeps the grouping of headers clean.